lib/rules/no-throw-literal.js

```diff
@@ -1,12 +1,50 @@
 /**
  * @fileoverview Rule to restrict what can be thrown as an exception.
  * @author Dieter Oberkofler
+ * @copyright 2015 Ian VanSchooten. All rights reserved.
  * @copyright 2015 Dieter Oberkofler. All rights reserved.
  */
 
 "use strict";
 
 //------------------------------------------------------------------------------
+// Helpers
+//------------------------------------------------------------------------------
+
+/**
+ * Determine if a node has a possiblity to be an Error object
+ * @param  {ASTNode}  node  ASTNode to check
+ * @returns {Boolean}       True if there is a chance it contains an Error obj
```

Comment:
In JavaScript, `Boolean` is an object while `boolean` is a primitive. The uppercase letter indicates the difference (same for string and number).